packages/fxa-auth-server/test/local/routes/subscriptions.js

```diff
@@ -531,19 +531,22 @@ describe('subscriptions', () => {
 
   describe('POST /oauth/subscriptions/reactivate', () => {
     it('should reactivate cancelled subscriptions', async () => {
-      const res = await runTest(
-        '/oauth/subscriptions/reactivate',
-        {
-          ...requestOptions,
-          method: 'POST',
-          payload: { subscriptionId: SUBSCRIPTION_ID_1 },
-        }
-      );
+      const res = await runTest('/oauth/subscriptions/reactivate', {
+        ...requestOptions,
+        method: 'POST',
+        payload: { subscriptionId: SUBSCRIPTION_ID_1 },
+      });
```

Comment:
Turns out the post-commit hook wasn't working when I did #1655, I only just fixed it now hence the fixing up here.